zerver/webhooks/harbor/doc.md

```diff
@@ -0,0 +1,18 @@
+Get Zulip notifications for your Harbor projects!
```

Comment:
Let's make `Harbor` here a link to the Harbor homepage?

Reply:
Definitely! Done in 37914bd34f712480d150021928cfcf6f421e51c9